### ISSUE TITLE
Setting different learning rate in different layer

### ISSUE THREAD

**Issue by zhly0** [GeorgeSeif/Transfer-Learning-Suite#3]
Hi，GeorgeSeif：
Thanks for your transfer learning code，if I want to setting different learning rate in different layer?Is keras
possible to do this ?
Thanks!
